docs/make_parameter_guideplot.py

```diff
@@ -27,15 +27,17 @@ def make_gasoil_plot(show=True, krgendanchor="sorg"):
     swl = 0.1
     sgcr = 0.2
     sorg = 0.2
+    sgro = 0.1
```

Comment:
Need sgro=sgro (for end-point scaling of krosgro(=KROR)

Reply:
That would make the plot hard to read (?), as it would not be evident where krosgro is attached.

Reply:
Could just add '=sgcr' in the annotation (and drop the sgcr ann..)?